frontend/src/utils/admin_irr.js

```diff
@@ -17,13 +17,17 @@ function drawHeatmap(response) {
     let all_zero = true;
     //don't draw the heatmap if there is no data
     let most_data = 0;
-    for (let i = 0; i < data.length; i++) {
```

Comment:
![Screen Shot 2021-08-18 at 10 16 42 AM](https://user-images.githubusercontent.com/10951569/129915533-ecb7e247-d48e-4572-94a9-5bfae60e5b07.png)

So I'm running into some rendering problems with this chart. Have you seen this before?